### DISCUSSION TITLE
How I can change my Profile Pic

### DISCUSSION THREAD

**Discussion by underhooddev** [community/community#70080] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I want to change profile to  gravatar.

**Comment by PriyanshuPz** [comment]
You can change gravatar by going to settings hovering on profile pic it will show you to invert profile pic to gravatar.
Just click on that.

**Comment by Akash1134** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
